# ANGUS 2019 advertisement

July 1 - July 13, 2019
University of California, Davis

[2018 materials: http://angus.readthedocs.io/en/2018/](http://angus.readthedocs.io/en/2018/)

This workshop is organized and hosted by Dr. C. Titus Brown and
Dr. Karen Word. Contact us at [dibsi.training@gmail.com](mailto:dibsi.training@gmail.com)!

**Lead instructors for 2019:**

* [Mike Lee](https://twitter.com/astrobiomike)
* [Rocio Martinez Nunez](https://kclpure.kcl.ac.uk/portal/en/persons/rocio-martinez-nunez(ce8b2695-8bb3-4430-bd65-4a415bd92bbe).html)
* [Sateesh Peri](https://twitter.com/perisateesh)
* [Taylor Reiter](https://twitter.com/ReiterTaylor)
* [Chissa Rivaldi](https://twitter.com/powerofcheez?lang=en)
* [Marian Schmidt](https://twitter.com/micro_marian)

**Confirmed lecturers for 2019:**

* [Erich Schwarz](https://scholar.google.com/citations?hl=en&user=VllBDvcAAAAJ&view_op=list_works&sortby=pubdate)

Registration has now closed.

Please also see [Frequently Asked Questions for 2019](https://hackmd.io/s/BkjZDqNNV).

This intensive two week summer course introduces attendees with a strong biology background to the practice of analyzing big shotgun sequencing data sets (Illumina, PacBio, and Nanopore). We introduce students to computational thinking and large-scale data analysis on UNIX platforms, and cover genome and transcriptome analysis. We also cover computational topics including R scripting, software installation with bioconda, cloud computing, and building efficient and automated workflows.

**No prior programming experience is required**, although bravery in the face of the unknown is necessary!  A year or more of graduate school in a
biological science is suggested. Faculty, postdocs, and research staff are more than welcome, as are researchers from industry.

The 2019 schedule of hours [is now available](SCHEDULE.html)!

We plan to run multiple independent classrooms of 20-30 participants each, with two lead instructors and 3-5 helpers per classroom. In 2019, we will provide both introductory and intermediate tracks to accomodate all levels of learners.

We strive to provide a welcoming and inclusive environment, and work hard to accomodate learner needs. This course supporst and encourages diversity by creating an environment in which people can communicate effectively and share their unique talents. This diversity of talents enriches the course by fueling creativity, innovation, and success!

## What will I learn if I attend?

Our goal for these two weeks is to get students to the point where they are ready to begin analyzing their own data on a computer cluster, and can work with help forums and other online resources to advance their own skills.

Students will gain practical experience in:

- bash shell and R scripting
- Cloud computing/Jetstream
- Installing and running many bioinformatics tools, including BLAST, bwa, samtools, 
- Querying mappings and evaluating assemblies
- Running workflows in the cloud and evaluating their results
- Estimating computational resources

We deliver and enhance these materials through live hands-on coding, group notes, and lots of friendly helpers!

Materials from previous courses are available at
  [http://angus.readthedocs.org/](http://angus.readthedocs.org/) under a Creative Commons/full use+reuse license.

You can read a blog post about the 2015 course here:
[http://ivory.idyll.org/blog/2015-small-batch.html](http://ivory.idyll.org/blog/2015-small-batch.html)

The course fee will be $950 for this workshop. On-campus housing for
the two weeks of the ANGUS course is $1005; housing includes breakfast
and dinner, but not lunch.

## What do people say about this workshop?

Quotes from previous attendees:

> “I would climb a mountain and share this awesome program with the rest of the world.”

> “We had great teachers and helpers. They made me feel comfortable in making mistakes and learn from them.”

> “You don’t have to be a programmer to use or do computational science, and that is awesome!"

> "I definitely broke through the difficult “starting” barrier, which was what I hoped to do. Now the techniques don’t seem so opaque and intimidating that I can’t work through them on my own.”

> "My entire outlook on computational biology changed. I am much more enthusiastic about continuing to learn and apply computational analyses to my research.”

## Computer requirements

You will need to bring a computer that can connect to wifi, and you should have a modern browser (Google Chrome or Safari or Firefox) installed.  No specific operating system is required.

We will use the [XSEDE Jetstream](https://jetstream-cloud.org/) academic cloud computing to execute data analysis for the workshop; all analysis will be done remotely.

## History

This workshop was run at Michigan State University's Kellogg Biological Station from 2010 to 2016, with support from the USDA and NIH (see [Funders](FUNDERS.html)).  Dr. Brown is the founding course director and ran the workshop from 2010-2015; Dr. Staton (UTK) and Dr. MacManes (UNH) were the 2016 course directors. In 2017 the course moved to UC Davis with Dr. Brown's lab, and we were able to expand to serve over 70 learners.

There are now over 350 alumni of the first 9 years!
